src/components/signin/components/Signin.jsx

```diff
@@ -73,14 +75,22 @@ const Signin = () => {
     const { formData, handleInputChange } = useInputFields();
     const dispatch = useDispatch();
     const { history } = useReactRouter();
-    const signinState = useSelector(state => state.signin);
+    const signinState = useSelector((state) => state.signin);
     return (
         <AuthLayout>
             <div className="p-signinPage">
+                <div className="p-authPage__switch">
+                    Not yet a user?{' '}
+                    <Link to="/signup" className="">
+                        Sign up
```

Comment:
here as well

Reply:
All of these were as a result of an extension I think, or a setting. Will revert all